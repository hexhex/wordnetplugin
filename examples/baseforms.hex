word("axes").
synonyms(P,S1,S,S2) :- &wordnet[23,W](P,S1,S,S2), word(W).

synonyms(1,1,"axe",1), 
synonyms(1,1,"ax",1), 
synonyms(2,1,"axe",1), 
synonyms(2,1,"ax",1), 
synonyms(2,2,"axe",2), 
synonyms(2,2,"ax",2)?
