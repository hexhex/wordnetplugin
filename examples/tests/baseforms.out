{synonyms(1,1,"axe",1), synonyms(1,1,"ax",1), synonyms(2,1,"axe",1), synonyms(2,1,"ax",1), synonyms(2,2,"axe",2), synonyms(2,2,"ax",2), word("axes")}
